src/entities/activity/ui/items/Date.tsx

```diff
@@ -1,12 +1,16 @@
+import Box from "@mui/material/Box"
+
 import { DateItemBase } from "~/shared/ui"
 
 type Props = {
   value?: string
 
   onValueChange: (value: string[]) => void
+
+  width?: string
 }
 
-export const DateItem = ({ value, onValueChange }: Props) => {
+export const DateItem = ({ value, onValueChange, width }: Props) => {
```

Comment:
Exported variable name and the file name should be the same. I suggest renaming the file

Reply:
Done